src/modules/functions.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-restricted-globals */
+// add button function
+let listTasks = [];
+if (localStorage.getItem('listTasks') !== null) {
+  listTasks = JSON.parse(localStorage.getItem('listTasks'));
+}
+// DOM manipulation
+const addbtn = document.querySelector('.add');
+const msg = document.querySelector('.msg');
+const txtField = document.getElementById('taskTextField');
```

Comment:
- [ ] As per the requirement below, please add a method for editing the task decriptions.
![image](https://github.com/ZabihullahNooriWardak/brandNewTodoApp/assets/91486791/32b42cc9-d329-45a2-91e1-0ec7b95687db)

Reply:
👏👏👏👏

You have correctly implemented the edit function.